README.md

```diff
@@ -128,7 +128,7 @@ job('upstreamJob') {
     }
 
     triggers {
-        githubPullRequest {
+        pullRequest {
```

Comment:
So, realizing perhaps this should be an issue.  I'm unable to use `githubPullRequest`, as exemplified in the README (see ticket description for the error I'm getting), but I did have luck with using `pullRequest`, though it is technically deprecated as seen [here](https://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.helpers.triggers.TriggerContext.pullRequest).